packages/rum-core/src/browser/htmlDomUtils.ts

```diff
@@ -43,3 +43,18 @@ export function forEachChildNodes(node: Node, callback: (child: Node) => void) {
 export function getParentNode(node: Node): Node | null {
   return isNodeShadowRoot(node) ? node.host : node.parentNode
 }
+
+/**
+ * Return the classList of an element or an array of classes if classList is not supported
+ *
+ * In cases where classList is not supported, such as in IE11 for SVG and MathML elements,
+ * we fallback to using element.getAttribute('class').
+ * We opt for element.getAttribute('class') over element.className because className returns an SVGAnimatedString for SVG elements.
+ */
+export function getClassList(element: Element): DOMTokenList | string[] {
```

Comment:
**💬 suggestion:** ‏put it in `polyfills`, as it is more a polyfill than an utility

Reply:
Do you think about moving it in the `core` package polyfills or to create another polyfills in `rum-core`?
I guess since Logs do need it would be creating one for rum-core?

Reply:
Creating a `polyfill` module in `rum-core` would be slightly more mindful when the SDK is consumed without treeshaking abilities!